biojava-structure/src/main/java/org/biojava/nbio/structure/symmetry/core/QuatSymmetryResults.java

```diff
@@ -111,14 +111,17 @@ public QuatSymmetryResults(List<SubunitCluster> clusters,
 	public List<SubunitCluster> getSubunitClusters() {
 		return Collections.unmodifiableList(clusters);
 	}
-	
+
 	/**
 	 * Returns the List of Subunits used to calculate symmetry.
 	 *
 	 * @return an unmodifiable view of the List
 	 */
 	public List<Subunit> getSubunits() {
-		return Collections.unmodifiableList(subunits);
+		if (subunits != null)
```

Comment:
Subunits should never be null. The bug is that some code is missing from the helical symmetry constructor. The right fix is to add the following code into line 101:

<pre>
 subunits = new ArrayList<Subunit>();
 for (SubunitCluster c : clusters) {
 	subunits.addAll(c.getSubunits());
 }
<pre>

Reply:
Thanks for the tip, now done in 96c6816366bbc7261e304aafee79c8a3ba04e